portality/api/v2/data_objects/application.py

```diff
@@ -0,0 +1,379 @@
+                "pissn": {"coerce": "unicode"},
+                "publication_time_weeks": {"coerce": "integer"},
+                "title": {"coerce": "unicode"}
+            },
+            "lists": {
+                "keywords": {"coerce": "unicode", "contains": "field"},
+                "language": {"coerce": "isolang_2letter", "contains": "field"},
+                "license": {"contains" : "object"},
+                "subject": {"contains": "object"}
+            },
+            "objects": [
+                "apc",
+                "article",
+                "copyright",
+                "deposit_policy",
+                "editorial",
+                "institution",
+                "other_charges",
+                "pid_scheme",
+                "plagiarism",
+                "preservation",
+                "publisher",
+                "ref",
+                "waiver"
+            ],
+            "structs": {
+                "apc": {
+                    "fields": {
+                        "url": {"coerce": "unicode"},
+                        "has_apc": {"coerce": "unicode"}
+                    },
+                    "lists": {
+                        "max": {"contains": "object"}
+                    },
+                    "structs": {
+                        "max": {
+                            "fields": {
+                                "currency": {"coerce": "unicode"},
+                                "price": {"coerce": "integer"}
+                            }
+                        }
+                    }
+                },
+                "article": {
+                    "fields": {
+                        "embedded_license": {"coerce": "bool"},
+                        "embedded_license_url": {"coerce": "unicode"},
+                        "orcid": {"coerce": "unicode"},
+                        "i4oc_open_citations": {"coerce": "unicode"}
+                    }
+                },
+                "copyright": {
+                    "fields": {
+                        "author_retains": {"coerce": "bool"},
+                        "url": {"coerce": "unicode"},
+                    }
+                },
+                "deposit_policy": {
+                    "fields": {
+                        "has_policy": {"coerce": "bool"},
+                        "is_registered": {"coerce": "bool"}
+                    },
+                    "lists": {
+                        "service": {"coerce": "unicode", "contains": "field"}
+                    }
+                },
+                "editorial": {
+                    "fields": {
+                        "review_url": {"coerce": "unicode"},
+                        "board_url": {"coerce": "unicode"}
+                    },
+                    "lists": {
+                        "review_process": {"contains" : "field", "coerce": "unicode",
+                                           "allowed_values": ["Editorial review", "Peer review", "Blind peer review",
```

Comment:
I've not specified allowed values in the main model struct, but perhaps we should.